scripts/check_frontend_coverage.py

```diff
@@ -145,6 +145,7 @@
     'editor-navigation.directive.ts',
     'editor-reloading-modal.controller.ts',
     'email-dashboard-data.service.ts',
+    'entity-creation.service.ts',
```

Comment:
I see that this file has a spec file, is it being fully covered?

Reply:
No, it's not, But I'll be removing this #9525 goes in, since I won't have to test skill creation here.

Reply:
Ok, thanks for clarifying!